breathecode/events/tests/urls/tests_academy_eventype_slug.py

```diff
@@ -38,6 +38,21 @@ def get_serializer(event_type, academy=None, city=None, data={}):
     }
 
 
+def get_put_serializer(event_type, data={}):
+
+    return {
+        'academy': event_type.academy,
+        'id': event_type.id,
+        'name': event_type.name,
+        'slug': event_type.slug,
+        'lang': event_type.lang,
+        'allow_shared_creation': event_type.allow_shared_creation,
+        'description': event_type.description,
+        'visibility_settings': event_type.visibility_settings,
```

Comment:
I saw `'visibility_settings'` is here, that means the put serializer is able to edit the visibility settings, I think that ain't right, also you shall to check post serializer, the endpoints related to `EventType` can't modify the visivility setting because here can bypass the validations